### ISSUE TITLE
Management of recording files

### ISSUE THREAD

**Issue by rrolf** [elan-ev/opencast-studio#18]
currently I see after the dialog to upload to Opencast or to download the videos. If I select upload I loose the option to download later.

Why not make this available permanently?

I am not sure how the recordings are managed internally. Does a recording overwrite the last recording? When is a recoding deleted: when I close the page, or the browser, when I start a new recording or never?

Instead of the save media button I would show a download of the videos in the main screen if they are available. If a history of old recordings is available I would show them all. And as we offer the videos as separate downloads the confusing multi-file download can be avoided.

**Comment by luniki** [comment]
That would work like the link to the gallery on Android?

![Screenshot_20191009-110109](https://user-images.githubusercontent.com/2993/66467547-e92af500-ea84-11e9-8989-3f5702c3b813.png)

**Comment by luniki** [comment]
I will refactor the "Save media" dialog into its own page. Some of the shortcomings will be addressed this way.

On providing media files "permanently": We will have to investigate on how much memory we get from the browser. Perhaps some kind of cloud drive could be another option as well.

**Comment by LukasKalbertodt** [comment]
It seems like browsers immediately throw away all `Blob` objects used by Studio once the user closes the tab. At least for Chrome on Ubuntu I could verify that: the blob url is accessible while the Studio tab is still open; the blob URL becomes inaccessible immediately after the Studio tab was closed; as this was a huge recording (2GB) I could observe that my free disk memory immediately increased by 2GB once I closed the Studio tab. So it doesn't seem possible to access past recordings.

About saving the recordings somewhere where we can access them later: there are certainly ways, but browsers never give any guarantee for how long that data is accessible. I don't think this is useful for users. 

We currently do a pretty good job at preventing the user from leaving accidentally when they have a recording. We also allow the user to download the recording (that way they have full control over it). Lastly, adding UI to manage past recordings would add a lot of complexity. 

So I will close this issue now.
